x-pack/plugin/src/test/resources/rest-api-spec/test/ml/delete_expired_data.yml

```diff
@@ -34,3 +56,96 @@ setup:
         body:  >
            { "timeout": "10h", "requests_per_second": 100000.0 }
   - match: { deleted: true}
+
+---
+"Test delete expired data with job id":
```

Comment:
What happens if somebody calls `_delete_expired_data` with a job that does not exist?